app/ui-react/syndesis/src/modules/integrations/components/editor/EditorStepsWithToolbar.tsx

```diff
@@ -0,0 +1,126 @@
+import { Connection, ConnectionOverview } from '@syndesis/models';
+import {
+  ConnectionsListView,
+  IActiveFilter,
+  IConnectionsListViewProps,
+  IFilterType,
+  ISortType,
+} from '@syndesis/ui';
+import { WithListViewToolbarHelpers } from '@syndesis/utils';
+import * as React from 'react';
+import { Translation } from 'react-i18next';
+import i18n from '../../../../i18n';
+import resolvers from '../../../connections/resolvers';
+import { EditorSteps, IEditorStepsProps } from './EditorSteps';
+import { IUIIntegrationStep } from './interfaces';
+
+function getFilteredAndSortedConnections(
+  steps: IUIIntegrationStep[],
+  activeFilters: IActiveFilter[],
+  currentSortType: ISortType,
+  isSortAscending: boolean
+) {
+  let filteredAndSortedConnections = steps;
+  activeFilters.forEach((filter: IActiveFilter) => {
+    const valueToLower = filter.value.toLowerCase();
+    filteredAndSortedConnections = filteredAndSortedConnections.filter(
+      (c: Connection) => c.name.toLowerCase().includes(valueToLower)
```

Comment:
This cast doesn’t feel right. Name should be a common property

Reply:
Yeah, I'd suggest continuing to rename these variables for this file, in this case it's `filteredAndSortedEditorSteps` not connections.  and then not `c`, maybe `editorStep` or `es`.